src/Uno.UI/Behaviors/VisibleBoundsPadding.cs

```diff
@@ -128,7 +128,7 @@ public static void SetPaddingMask(DependencyObject obj, PaddingMask value)
 			=> obj.SetValue(PaddingMaskProperty, value);
 
 		public static DependencyProperty PaddingMaskProperty { get; } =
-			DependencyProperty.RegisterAttached("PaddingMask", typeof(PaddingMask), typeof(_VisibleBoundsPadding), new PropertyMetadata(PaddingMask.None, OnIsPaddingMaskChanged));
+			DependencyProperty.RegisterAttached("PaddingMask", typeof(PaddingMask), typeof(_VisibleBoundsPadding), new FrameworkPropertyMetadata(PaddingMask.None, OnIsPaddingMaskChanged));
```

Comment:
This can't be changed because it's built by UWP. (And the Toolkit is not really 'internal' to the framework anyway.)